### PR TITLE
Adding 'resize' action to resize an OpenStack instance using the

### PR DESCRIPTION
**Pull request by pablomartinez**
os_server_action cloud module.

##### SUMMARY

Adding the `resize` action to the OpenStack cloud modules as part of the os_server_action module. This will resize an OpenStack instance.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

os_server_action:resize
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/openstack/os_server_action.py [hunk 1]
```diff
@@ -239,6 +247,39 @@ def main():
...
+            if flavor is None:
+                module.fail_json(msg="Flavor does not exist")
+            if flavor.id == server.flavor.id:
+                module.exit_json(changed=False, debug='Same flavor as the server')
+
+            cloud.compute.post(
+                _action_url(server.id),
+                json={'resize': {'flavorRef': flavor.id}})
+            # Waiting is not conditional as we need to do a second task
+            for count in sdk.utils.iterate_timeout(
```
Comment (cloudnull) [inline lib/ansible/modules/cloud/openstack/os_server_action.py:264]: NIT: This is a hanging object, if its not going to be used lets set it to `_`.

``` python
 for _ in sdk.utils.iterate_timeout(
    timeout,
    "Timeout waiting for server to complete %s" % action):
```

**Review by cloudnull — CHANGES_REQUESTED** [review]
one nit but otherwise looks good to me.

File: lib/ansible/modules/cloud/openstack/os_server_action.py (hunk 1 above)
Comment (pablomartinez) [reply lib/ansible/modules/cloud/openstack/os_server_action.py:264]: Sorry for the delay, I've just commited a change that fixes that nit.

**Comment by pablomartinez** [comment]
Is there anything that can be done with this PR?

**Review by cloudnull — APPROVED** [review]
shipit

**Comment by sshnaidm** [comment]
Thanks for submitting patch for Openstack Ansible modules!
We moved Openstack Ansible modules to Openstack repositories.
Next patches should be submitted not with Ansible Github but with
Openstack Gerrit: https://review.opendev.org/#/q/project:openstack/ansible-collections-openstack
Please submit your code there from now.
Thanks for your contribution and sorry for inconvienience.

**Comment by Akasurde** [comment]
@pablomartinez Thanks for the contribution. Closing as per above.

**Pull request closed without merging**